plugins/ekuiper/ekuiper.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
@fengzeroz  The dashboard won't send settings according to this schema, and neuron won't turn the node state to ready, which is problemaitc.